builder/tests/test_views.py

```diff
@@ -40,10 +40,15 @@ def test_version_from_scratch(client, version_from_scratch):
 
 
 def test_search(client, draft_with_some_searches):
-    rsp = client.get(draft_with_some_searches.get_builder_search_url("arthritis"))
+    slug = "arthritis"
+    for search in draft_with_some_searches.searches.all():
+        if search.term == slug:
+            search_id = search.id
```

Comment:
```suggestion
    search_id = draft_with_some_searches.searches.filter(term=slug).first().id
```

Think using a queryset filter rather than iteration is more idiomatic and marginally more readable.

Reply:
That's a lot better thanks. I'm still learning the django model queries.